SimpleNtuplizer/plugins/SNtup_PFClusters.cc

```diff
@@ -33,19 +30,17 @@ void SimpleNtuplizer::setPFVariables(const edm::Event& iEvent,
     for (reco::PFClusterCollection::const_iterator pfc=(*clustersH).begin(); pfc!=(*clustersH).end(); pfc++){
       
       ///energy
-      double energy = (*pfc).energy();
-      //cout<<"Energy is "<<energy<<endl;
       
-      clusE_.push_back(energy);
+      clusE_pf = pfc->energy();
```

Comment:
Is this the corrected energy? We need both: the corrected and the uncorrected energy. The first to compare, the second to use as input for the correction.